spec/app_support/reservations/duration_change_validations_spec.rb

```diff
@@ -68,7 +68,7 @@
 
               expect(validator).not_to be_valid
               expect(reservation.errors.full_messages)
-                .to include("Reserve Start cannot change once the reservation has started")
+                .to include("Reserve start at cannot change once the reservation has started ")
```

Comment:
This has a bit of a smell to me - maybe we need to use `messages` instead of `to_a`?  The upgrade should not have side effects like this if we can avoid it